pom.xml

```diff
@@ -51,6 +51,20 @@
             <version>1.7.26</version>
             <scope>test</scope>
         </dependency>
+        
+        <!-- Added for Unit testing the UI -->
+        <dependency>
+            <groupId>org.assertj</groupId>
```

Comment:
This dependency may not be needed if you use:
org.junit.Assert* 
 *assertEquals
 *assertTrue
 *assertFalse

Less dependencies used the better :)